starter-code/src/app/components/log-form/log-form.component.html

```diff
@@ -0,0 +1,38 @@
+<p>
+  log-form works!
+</p>
+
+<form (ngSubmit)="submitForm(form)" #form="ngForm" novalidate>
+
+  <div class="field" [ngClass]="{'has-error': feedbackEnabled && personField.errors}">
+    <label>person</label>
+    <input type="text" name="person" [(ngModel)]="person" #personField="ngModel" required minlength="6" [disabled]="processing"/>
+    <div *ngIf="feedbackEnabled && personField.errors">
+      <p *ngIf="personField.errors.required" class="validation">person required</p>
+      <p *ngIf="personField.errors.minlength" class="validation">person is too short</p>
+    </div>
+  </div>
+
+  <div class="field" [ngClass]="{'has-error': feedbackEnabled && messageField.errors}">
+    <label>message</label>
+    <input type="message" name="message" [(ngModel)]="message" #messageField="ngModel" required minlength="6"  [disabled]="processing"/>
+    <div *ngIf="feedbackEnabled && messageField.errors">
+      <p *ngIf="messageField.errors.required" class="validation">message required</p>
+      <p *ngIf="messageField.errors.minlength" class="validation">message is too short</p>
+    </div>
+  </div>
+
+  <div class="field submit">
+    <button type="submit" (click)="handleAccessItem()" [disabled]="processing">Login</button>
```

Comment:
Why are you binding a method to the click when you have a ngSubmit in the form (line 5).